framework/PageToolbar/PageToolbarFilter.tsx

```diff
@@ -281,35 +272,24 @@ function ToolbarFilterComponent(props: {
 
     case ToolbarFilterType.SingleSelect:
       return (
-        <ToolbarSelectFilter
+        <ToolbarSingleSelectFilter
           id={props.id ?? filter.key}
-          values={values}
-          addFilter={addFilter}
-          removeFilter={removeFilter}
+          placeholder={filter.placeholder ?? 'TODO'}
```

Comment:
Could we possibly provide some additional context in this situation? It would be great if the TODO could include enough information so that another person could comfortably carry out the task if needed, or even reference a related issue.

Reply:
fixed thanks!